plugins/tomcat/src/main/java/com/navercorp/pinpoint/plugin/tomcat/interceptor/RequestStartAsyncInterceptor.java

```diff
@@ -72,8 +74,9 @@ public void after(Object target, Object[] args, Object result, Throwable throwab
             final SpanEventRecorder recorder = trace.currentSpanEventRecorder();
             if (validate(target, result, throwable)) {
                 com.navercorp.pinpoint.bootstrap.context.AsyncContext nextAsyncContext = recorder.recordNextAsyncContext(true);
-                if (args != null && args.length > 1 && args[0] instanceof AsyncContextAccessor) {
-                    ((AsyncContextAccessor) args[0])._$PINPOINT$_setAsyncContext(nextAsyncContext);
+                Object asyncContextAccessor = ArrayUtils.get(args, 0);
+                if (asyncContextAccessor instanceof AsyncContextAccessor) {
+                    ((AsyncContextAccessor) asyncContextAccessor)._$PINPOINT$_setAsyncContext(nextAsyncContext);
```

Comment:
When args has a size of 1, the old code is not executed, but changed code is going to be executed.
I don't know if the size of args can be 1.